src/components/Elements/DropZone.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+import droppable from '../../behaviors/droppable';
+
+class DropZone extends Component {
```

Comment:
Are we linting for react? If so, are we linting to prefer pure functions over Components? This would be a candidate for function with no state and no methods other than render.